airbyte-integrations/connectors/destination-glide/destination_glide/destination.py

```diff
@@ -95,7 +95,7 @@ def create_table_client_for_stream(stream_name, columns):
             columns = []
             properties = configured_stream.stream.json_schema["properties"]
             for (prop_name, prop) in properties.items():
-                prop_type = prop["type"]
+                prop_type = prop["type"] if "type" in prop else ""
```

Comment:
So we're replacing a `nil` value in previous behavior with an empty string - how does that affect Airbyte functionality?

Reply:
I imagine it was crashing before, now `prop_type` will be set to "". 

```python
prop_type = prop["type"]
```
would have raised a key exception before if it wasn't defined.

Reply:
So, it looks like we will treat it as a string:

https://github.com/glideapps/airbyte/blob/894856cd86df6b9ca6a1ce1b77320b5630c04e60/airbyte-integrations/connectors/destination-glide/destination_glide/destination.py#L53

Reply:
Yep, we always fallback to that. That is expected behavior for airbyte destinations.